test/functional/p2p_leak.py

```diff
@@ -96,7 +98,11 @@ def set_test_params(self):
     def run_test(self):
         no_version_bannode = self.nodes[0].add_p2p_connection(CNodeNoVersionBan(), send_version=False, wait_for_verack=False)
         no_version_idlenode = self.nodes[0].add_p2p_connection(CNodeNoVersionIdle(), send_version=False, wait_for_verack=False)
-        no_verack_idlenode = self.nodes[0].add_p2p_connection(CNodeNoVerackIdle())
+        no_verack_idlenode = self.nodes[0].add_p2p_connection(CNodeNoVerackIdle(), wait_for_verack=False)
+
+        # Wait until we got the verack in response to the version. Though, don't wait for the other node to receive the
+        # verack, since we never sent one
```

Comment:
non-blocking suggestion (line wrap and content) here and p2p_timeouts.py::50:
```
        # Wait until we get the verack in response to the version. Do not wait
        # for the other node to receive the verack, since we never sent one.
```

Reply:
Most terminal sizes are 120 by default, also most review tools as well as GitHub web. So my personal preference is 120. I think I will leave it as is for now.